src/hooks/use-sync-states-progress-info.ts

```diff
@@ -182,6 +200,36 @@ export function useSyncStatesProgressInfo() {
 		[ __ ]
 	);
 
+	const getPushStatusWithProgress = useCallback(
+		( status: PushStateProgressInfo, response: ImportResponse ) => {
+			if ( status.key === 'importing' ) {
+				const backupStep = 10;
+				const archiveInitialValue = IMPORTING_INITIAL_VALUE + backupStep;
+				const archiveStep = IMPORTING_TO_FINISHED_STEP - backupStep;
```

Comment:
Probably we need need a different constant for `IMPORTING_TO_FINISHED_STEP`, which is also used on `getPullStatusWithProgress`.